internal/cli/atlas_measurements_process.go

```diff
@@ -0,0 +1,135 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type atlasMeasurementsProcessOpts struct {
+	*globalOpts
+	pageNum         int
+	itemsPerPage    int
+	host            string
+	granularity     string
+	period          string
+	start           string
+	end             string
+	measurementType string
+	store           store.ProcessMeasurementLister
+}
+
+type hostInfo struct {
+	hostName string
+	port     int
+}
+
+func (opts *atlasMeasurementsProcessOpts) init() error {
+	if opts.ProjectID() == "" {
+		return errMissingProjectID
+	}
+
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *atlasMeasurementsProcessOpts) Run() error {
+	listOpts := opts.newProcessMeasurementListOptions()
+
+	hostInfo, err := opts.newHostInfo()
+	if err != nil {
+		return err
+	}
+
+	result, err := opts.store.ListProcessMeasurements(opts.ProjectID(), hostInfo.hostName, hostInfo.port, listOpts)
+
+	if err != nil {
+		return err
+	}
+
+	return json.PrettyPrint(result)
+}
+
+func (opts *atlasMeasurementsProcessOpts) newHostInfo() (*hostInfo, error) {
+	host := strings.SplitN(opts.host, ":", -1)
+	if len(host) != 2 {
+		return nil, errors.New("the host information must be in the format host:port")
+	}
+
+	port, err := strconv.Atoi(host[1])
+	if err != nil {
+		return nil, err
+	}
+
+	hostInfo := &hostInfo{
+		hostName: host[0],
+		port:     port,
+	}
+	return hostInfo, nil
+}
+
+func (opts *atlasMeasurementsProcessOpts) newProcessMeasurementListOptions() *atlas.ProcessMeasurementListOptions {
+	return &atlas.ProcessMeasurementListOptions{
+		ListOptions: &atlas.ListOptions{
+			PageNum:      opts.pageNum,
+			ItemsPerPage: opts.itemsPerPage,
+		},
+		Granularity: opts.granularity,
+		Period:      opts.period,
+		Start:       opts.start,
+		End:         opts.end,
+		M:           opts.measurementType,
+	}
+}
+
+// mongocli atlas measurements process(es) host:port [--granularity granularity] [--period period] [--start start] [--end end] [--type type][--projectId projectId]
+func AtlasMeasurementsProcessBuilder() *cobra.Command {
+	opts := &atlasMeasurementsProcessOpts{
+		globalOpts: newGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Use:   "process",
+		Short: description.ProcessMeasurements,
+		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.init()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.host = args[0]
```

Comment:
[nit] depending on how you handle my above changes (function return string:int vs newHostInfo) you could change this to do that process here since you have no use for `opts.host` rather than to parse it later
with this I mean you could fo
```go
opt.host, opt.port, err = opts.getHostAndPort(args[0])
```
Or
```go
opt.hostInfo, err = newHostInfo(args[0])
```